project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.15
```

Comment:
probably not – I think IntelliJ makes these, and I've committed it accidentally.